[improve][doc] SEO for Tiered Storage (#684)
apache/pulsar-site: docs/tiered-storage-overview.md

id: tiered-storage-overview
title: Overview of tiered storage
sidebar_label: "Overview"

> description: Get a comprehensive understanding of concepts and install methods of Pulsar tiered storage.

Reviewer: ```suggestion
description: Get a comprehensive understanding of use cases, installation methods, and working principles of Pulsar tiered storage.
```

## How to install tiered storage offloaders?

Pulsar releases a separate binary distribution, containing the tiered storage offloaders. To enable those offloaders, you need to complete the following steps.

> 1. download the offloaders tarball release

Reviewer: ```suggestion
1. Download the offloaders tarball release.
```

```bash
wget pulsar:offloader_release_url
```

> 2. Untar the offloaders package and copy the offloaders as `offloaders` in the pulsar directory

Reviewer: ```suggestion
2. Untar the offloaders package and copy the offloaders as `offloaders` in the pulsar directory.
```
